influxdb3_write/src/wal.rs

```diff
@@ -347,9 +353,19 @@ impl WalSegmentWriter for WalSegmentWriterNoopImpl {
         self.segment_id
     }
 
-    fn write_batch(&mut self, _ops: Vec<WalOp>) -> Result<SequenceNumber> {
+    fn bytes_written(&self) -> u64 {
+        println!("getting bytes_written");
```

Comment:
Can we switch these `println!` to traces if they are needed? There is one in `fn write_batch` as well.